docs/guide/core-concepts/vite.md

```diff
@@ -0,0 +1,40 @@
+# Vite
+
+WXT uses [Vite](https://vitejs.dev/) under the hood to bundle your extension. As such, all of [Vite's features](https://vitejs.dev/guide/features.html) are available when using WXT.
+
+WXT orchastrates several different Vite builds to bundle your extension. For more info, read [How WXT Works](/guide/resources/how-wxt-works).
+
+## Change Vite Config
+
+You can change Vite's config via the `wxt.config.ts` file:
+
+```ts
+// wxt.config.ts
+export default defineConfig({
+  vite: () => ({
+    // Override config here, same as `defineConfig({ ... })`
+    // inside vite.config.ts files
+  }),
+});
+```
+
+## Add Vite Plugins
+
+To add a plugin, install it and add it to the config:
+
+```ts
+// wxt.config.ts
+export default defineConfig({
+  vite: () => ({
+    plugisn: [
```

Comment:
```suggestion
    plugins: [
```

Reply:
Thank you!

Reply:
No worries.
I'm actually going through this new WXT documentation
I'll let you know if I find anything out of place